subprojects/packagefiles/openpace/meson.build

```diff
@@ -0,0 +1,139 @@
+project('openpace', 'c',
+        version: '1.1.3',
+        license: 'GPL-3.0-only',
+        default_options: 'c_winlibs=ws2_32.lib',
+        meson_version: '>= 1.3.0'
+       )
+
+modfs = import('fs')
+
+absolute_sysconfdir = (modfs.is_absolute(get_option('sysconfdir'))? '' : get_option('prefix')) / get_option('sysconfdir')
+
+conf = configuration_data()
+
+cv_certificates_path = get_option('cv_certificates_path')
+if cv_certificates_path == ''
+    cv_certificates_path = absolute_sysconfdir / 'eac' / 'cvc'
+endif
+conf.set_quoted('CVCDIR', cv_certificates_path)
+
+x509_certificates_path = get_option('x509_certificates_path')
+if x509_certificates_path == ''
+    x509_certificates_path = absolute_sysconfdir / 'eac' / 'x509'
+endif
+conf.set_quoted('X509DIR', x509_certificates_path)
+
+depopenssl = dependency('openssl', version: '>= 1.0.2', required: false)
```

Comment:
can we set 1.1.1 minimum or even 3.0.0

Reply:
Indeed, this is what we agreed on already
https://github.com/OpenSC/OpenSC/blob/cc14481a68941f14e12a82ad8d6c10773ab55ac5/configure.ac#L25